### ISSUE TITLE
Suggested changes

### ISSUE THREAD

**Issue by vince62s** [OpenNMT/nmt-benchmark#2]
Guys,

On the main results page, when you show one system and the results per test set (eg from 1 to 6)
You should add a line which qualify the domain "News, Legal, ..."
you have this info since it appears in the testsets page.

Also, you may want to rename these domains.

My guess is that generic_xxxx is NOT a "News" domain. If this is a held-out test set from the training data you should specify "In-domain" as the domain.

The term "Legal" for some old testset like 2006-2008 is a bit misleading.
I really think at that time people were working on Europarl for Data and testset.
So Legal is in fact "Europarl" or again could be considered as "In-domain"

If you take the example of French to spanish. Sets 1 and 7 are clearly in-domain and we need to show this.
For 2 to 5 it "News"

All of this would clarify much more the results presentation for external viewers.

Cheers,
Vincent
